src/main/java/org/folio/circulation/domain/representations/DeclareItemLostRequest.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.failedValidation;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+
+public class DeclareItemLostRequest {
+  private static final String COMMENT = "comment";
+  private static final String DECLARED_LOST_DATETIME  = "declaredLostDateTime";
+
+  private final String comment;
+  private final DateTime declaredLostDateTime;
+  private final String loanId;
+
+  private DeclareItemLostRequest(String comment, DateTime declaredLostDateTime,
+    String loanId) {
+    this.comment = comment;
+    this.declaredLostDateTime = declaredLostDateTime;
+    this.loanId = loanId;
+  }
+
```

Comment:
Personally, I’d order the parameters / fields etc for precedence. To me, the `loanId` is the most important aspect, then the date (though this appears contentious) and then the comment.

This is minor, it can easily be changed later.

Reply:
Done.